cfn-resources/cloud-backup-snapshot-export-bucket/test/add-policy.json

```diff
@@ -9,7 +9,7 @@
       "Action": "sts:AssumeRole",
       "Condition": {
         "StringEquals": {
-          "sts:ExternalId": "3aa195a5-ecfd-4ed3-9a9f-afe1275067fa"
+          "sts:ExternalId": "6fb95014-fb21-42ed-a2c1-628862460b6a"
```

Comment:
[q] Where does this value come from? Is it a read resource that must exist in our testing env?

I'm concerned about the test being brittle if the underlying resource is destroyed.

Reply:
I deleted the values, These are filled by ./test/cfn-test-create-inputs.sh script during test.